### ISSUE TITLE
Problem with Pkg-config ?

### ISSUE THREAD

**Issue by romainlebreton** [linbox-team/fflas-ffpack#35]
Configure in fflas with --enable-optimization (and maybe also without optimization ) does not work. Pkg-config finds givaro.pc but it seems that the include path for headers is not set.

For instance, the optimization program is compiled with 

```
g++ -o conftest -I. -I.. -I/home/lebreton/lib/fflas-ffpack -I/home/lebreton/lib/fflas-ffpack/fflas-ffpack  
-std=gnu++11 -msse4.1 -mfma -mavx2 -fabi-version=6 -O2  -Wall -DNDEBUG -UFFLASFFPACK_DEBUG
 -O0 -Wall -g -DDEBUG -DGIVARO_DEBUG -std=gnu++11 -mmmx -mpopcnt -msse -msse2 -msse3 
-msse4.1 -msse4.2 -mavx -mavx2 -mfma -mbmi -mbmi2 -mmmx -mpopcnt -msse -msse2 -msse3 
-msse4.1 -msse4.2 -mavx -mavx2 -mfma -mbmi -mbmi2 -mfpmath=sse -fabi-version=6  
-D__FFLASFFPACK_HAVE_CBLAS -fopenmp -DFLTTYPE=Givaro::Modular<double> 
-DOPTIMISATION_MODE   conftest.cpp  -lopenblas -L/home/lebreton/lib/givaro/myinstall/lib 
-lgivaro -lgmpxx -lgmp >&5
```

After I add `-I/home/lebreton/lib/givaro/mysintall/include`, I get another error 

> error: #error "NDEBUG and DEBUG both defined"

**Comment by ClementPernet** [comment]
Ok, I mistakingly forgot to add -i@includedir@ in givaro.pc.in (and in fflas-ffpack too!).
This is now fixed from 79f6e6de287111e38b524815ed0a1589ce4e2261 (givaro repo)

**Comment by romainlebreton** [comment]
Thanks, it works now.
